src/modules/rewards/utils/stakeRewardUtilities.ts

```diff
@@ -0,0 +1,24 @@
+import { RewardsStakeParams } from 'queries';
+
+export const sortByIndexNumber = (a: any, b: any) => {
+  const numA = parseInt(a.index?.split('-').pop() || '0', 10);
+  const numB = parseInt(b.index?.split('-').pop() || '0', 10);
+  return numA - numB;
+};
```

Comment:
```suggestion
export const sortByIndexNumber = (a: Type, b: Type) => {
  const numA = parseInt(a.index?.split('-').pop() || '0', 10);
  const numB = parseInt(b.index?.split('-').pop() || '0', 10);
  return numA - numB;
};
```

User types and please write doc string comments with proper explantion.